### PR TITLE
61 w6 1b repository pattern

### PR DESCRIPTION
**Pull request by jfho**

### CODE REVIEW

File: src/Chirp.Razor/CheepRepository.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+using Microsoft.EntityFrameworkCore;
+using ViewModel;
+using Chirp.Models;
+
+public interface ICheepRepository
+{
+    //public List<CheepViewModel> GetCheeps();
```
Comment (Mardesol) [inline src/Chirp.Razor/CheepRepository.cs:7]: Do we want to keep this around? :)

**Review by Mardesol — CHANGES_REQUESTED** [review]

File: src/Chirp.Razor/CheepRepository.cs (hunk 1 above)
Comment (Niklas-ASA) [reply src/Chirp.Razor/CheepRepository.cs:7]: I will remove this

**Review by Mardesol — APPROVED** [review]
Good work!

**Review by dagruneir — APPROVED** [review]
look gooood

**Pull request merged**